lib/options.js

```diff
@@ -16,7 +17,8 @@ const analyze = {
     /Index of/,
     /Listing of/
   ],
-  logger: defaultLogger
+  logger: defaultLogger,
+  concurrency: cpus().length
```

Comment:
Plunger is not really cpu-intensive.
Why not the same value as maxSockets?

Reply:
`cpus().length` is probably not ideal, but I had to find a default 😄

I dont think that `maxSockets` and this serve the same purpose. `maxSockets` is so we don’t overload people’s servers with too many requests to large files.

This is more like a combination of cpu (not much), memory management, fs reads (and some writes), subprocesses (unar should be heavy on all those things). I’m not sure what to pick as a default value, I thought cpus().length was not too bad.